packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -65,7 +66,7 @@ class AddOrManageAccounts extends React.Component {
             } else {
                 // Add Crypto Account
                 this.props
-                    .createCryptoAccount(value)
+                    .createCryptoAccount(value, is_deriv_crypto)
```

Comment:
Possible to remove the argument here I think?

```suggestion
                    .createCryptoAccount(value, this.context.is_deriv_crypto)
```